app/ui/labels.py

```diff
@@ -222,6 +222,10 @@ def create_label(self, grouped: str, is_dynamic: bool):
         st.title("New Label")
         group = None
         rank = None
+        name = None
+
+        if not is_dynamic:
+            name = st.text_input(key="NAME", label="Label Name")
```

Comment:
I don't think we have strict guidance on this, but what was your thinking with this change?

For grouped labels, I think of it as a hierarchy: `group > label`. It makes more sense to me, when we're creating a grouped label that group comes first, and then name. WDYT?

Reply:
Agree with @joshelser, it does not feel natural to have label first. Especially when the second label in the group is created and group name is greyed out ... looks and feels weird.

Reply:
I based the ordering off of the old ui in the native app. The order there was:
```
label name
group name -- rank
condition
```
that said, I'm open to suggestions.